### DISCUSSION TITLE
Do you plan to patch Version 7 for the CRLF vulnerability ?

### DISCUSSION THREAD

**Discussion by simon-knu** [reactiveui/refit#1922] [General]
Hello ! 

Do you plan to patch version 7 with this vulnerability ? @glennawatson @ChrisPulman 
https://github.com/reactiveui/refit/pull/1834

I need to go in prod soon and I want to avoid a major change before that so I want to know if you plan to patch the version 7 for this vulnerability 

Best regards

**Comment by glennawatson** [comment]
I did a release the other day. I believe that pr is part of that

**Reply by glennawatson** [reply]
https://github.com/reactiveui/refit/releases/tag/8.0.0

**Comment by glennawatson** [comment]
We also don't patch old releases. Always just version and move on. So there won't be an another 7 release. 

Probably your best move is grab the repo. Go to the last version 7 tag and cherry pick the fix in. 

Get your release out. 

Then upgrade to 8. 

I'd be willing to do a once off for you of a 7.0 release if you can do the work of attempting to do a pull request to a 7.x branch.

**Comment by simon-knu** [comment]
Oh ok, I understand, thank you for the quick response !

I tried to do the job you describe here : https://github.com/reactiveui/refit/pull/1923 but since I can't create a branch based on the tag 7.2.1 in this repo, I don't think it's possible

**Reply by simon-knu** [reply]
But if there is a branch based on 7.2.1 here, I can' definitely do that !

**Reply by glennawatson** [reply]
> But if there is a branch based on 7.2.1 here, I can' definitely do that !

Let me see what I can do for you

**Reply by glennawatson** [reply]
https://github.com/reactiveui/refit/tree/7.2.1-hotfix

**Reply by simon-knu** [reply]
Thank you !!! Sorry for the delay, the PR is created !

**Reply by simon-knu** [reply]
Wait, did you create the branch based on the tag 7.2.1 ?
It's seems the branch is up to date to main

**Reply by TimothyMakkison** [reply]
IIrc 7.2.1 was released on commit f68cc31 or 29e0e1c on September 17, see #1819

**Reply by glennawatson** [reply]
We know @TimothyMakkison -- he just wants a hotfix based on 7 to get one update in there, given @simon-knu has contributed in the past I'm doing a one time gesture to help him out

**Reply by simon-knu** [reply]
@glennawatson Could you confirm that it the branch [7.2.1-hotfix](https://github.com/reactiveui/refit/tree/7.2.1-hotfix) is based on the tag 7.2.1 ? Because it does not seems to be the case

**Reply by glennawatson** [reply]
7.2.22 is released with that vulnerability patch, some updated nuget packages which were vulnerable.

**Reply by simon-knu** [reply]
thank you !!!

**Reply by simon-knu** [reply]
For info @glennawatson, the quick fix didn't help :/. The 7.2.22 is tagged as containing a vulnerability, I will try to update to 8.0 and test to see if things break

Thank you for the try tho

**Reply by glennawatson** [reply]
Yeah the cve has flagged any release 7.x. But the vulnerability is fixed.

**Reply by ChrisPulman** [reply]
I can see if I can update the CVE

**Reply by glennawatson** [reply]
I've pushed an update to that database so hopefully that vulnerability marker will be removed.

**Comment by ChrisPulman** [comment]
The major version bump was mostly due to the CVE, there was a few alterations made to the API too but shouldn't be a big issue for updates, bug fixes lead to the changes, so the results may differ slightly depending upon your use case.

**Comment by simon-knu** [comment]
thank you for your quick reactions guys !
